src/app/public/modules/file-attachment/fixtures/file-drop-control-template.component.fixture.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Component,
+  ViewChild
+} from '@angular/core';
+
+import {
+  SkyFileDropControlComponent
+} from '../file-drop-control.component';
+
+import {
+  SkyFileDropValue
+} from '../types/file-drop-value';
+
+@Component({
+  selector: 'file-drop-control-template-test',
+  templateUrl: './file-drop-control-template.component.fixture.html'
+})
+export class FileDropControlTemplateTestComponent {
+
+  public attachment: SkyFileDropValue;
+
+  public required: boolean = false;
```

Comment:
It looks like `required` and `showLabel` aren't being used?

Reply:
Removed